eth_tester/backends/pyevm/main.py

```diff
@@ -259,7 +276,13 @@ def _get_vm_for_block_number(chain, block_number):
 class PyEVMBackend(BaseChainBackend):
     chain = None
 
-    def __init__(self, genesis_parameters=None, genesis_state=None, vm_configuration=None):
+    def __init__(
+        self,
+        genesis_parameters=None,
+        genesis_state=None,
```

Comment:
I noticed none of these params are documented besides `vm_configuration`...

If it would be helpful, I can document this method's params and maybe a bit to the README about the `mnemonic` usage?

Reply:
That would be very helpful, thanks!